src/PharoThings-Devices-HD44780.package/PotHD44780Controller.class/instance/setCursorAtRow.col..st

```diff
@@ -0,0 +1,5 @@
+commands
+setCursorAtRow: anInteger col: anInteger2
```

Comment:
In Smalltalk style it is better to use full name for words and use meaningful names for arguments:
```Smalltalk
setCursorAtRow: row column: column   
```